src/Utility.ts

```diff
@@ -212,6 +212,6 @@ export namespace Utility {
         if (config) {
             javaPath = config.get<string>('home');
         }
-        return javaPath ? javaPath + '/bin/java' : 'java';
+        return javaPath ? "\"" + javaPath + '/bin/java' + "\"" : 'java';
```

Comment:
should use `path.join(javaPath, "bin", "java")` to calculate file path, otherwise it's possible not working with some terminals under Windows, which use `\` as separator".

As for the whitespaces, quoting with `"` doesn't always solve the problem based on the default terminal you are using.  If the issue is valid, it's probably related to the way you spawn it, and we should fix it there.

Reply:
Oh, thanks a lot!
I close this pull request.